### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ Create New Web App_Keyboard: When New key is added into the table, keyboard user is able to edit it only for the first time unlike mouse users

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3346]
[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS2.1.1

**Repro Steps:** 
1) Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Within Eclipse's Project Explorer view, expand Newly created dynamic web project. Right-click WebContent, click New-->JSP File
6) Tab till Next button and press enter
7) Select New JSP File (html) and press Finish
8) Open JSP file and write any code in the editor
9) In Project Explorer view, right-click your project, choose Azure-->Publish as Azure Web App
10) Tab till Create button and press enter
11) Tab till ‘New’ button and press enter. Edit the key and enter any value
12) Verify if users can be able to edit the key after the first time

**Actual Result:** 
When New key is added into the table, keyboard user can edit it only for the first time unlike mouse users

**Expected Result:** 
Keyboard users should be able to edit the key as many times as they want

**User Impact:** 
Keyboard users may not be able to change the key more than once if the operation is not available to them

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[Bug_16_user cannot edit teh key for more than once.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451281/Bug_16_user.cannot.edit.teh.key.for.more.than.once.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati  this issue has been fixed in #3511, you may find the build [here](https://vscjavaci.cloudapp.net/job/Azure_Toolkits_for_Java/job/AzureToolsManager_PR/3775/Azure/processDownloadRequest/artifacts/WindowsAzurePlugin4EJ.PR.3775.09-09-2019.zip), could you please help us verify it? Thanks!

**Comment by Tanushagottipati** [comment]
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

Issue fixed by downloading above build. hence closing the bug and attached latest video for reference
[Bug fixed for create web app.pptx](https://github.com/microsoft/azure-tools-for-java/files/3590415/Bug.fixed.for.create.web.app.pptx)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749749
